lightning/src/onion_message/messenger.rs

```diff
@@ -243,6 +244,89 @@ impl OnionMessageRecipient {
 	}
 }
 
+pub trait RespondFunction<T: OnionMessageContents>{
+	fn respond(&self, response: T, reply_path: BlindedPath);
+}
+
+impl<F, T> RespondFunction<T> for F
+where
+T: OnionMessageContents,
+F: Fn(T, BlindedPath)
```

Comment:
Please indent these here and elsewhere.

Reply:
Addressed in [2806122](https://github.com/lightningdevkit/rust-lightning/pull/2907/commits/2806122f65d57d3a6a91591aff6e9ccb7590af13)
Thanks!